backend/workflow_manager/workflow/execution.py

```diff
@@ -105,13 +103,6 @@ def __init__(
 
         self.compilation_result = self.compile_workflow(execution_id=self.execution_id)
 
-    def _initiate_api_execution(
```

Comment:
Why it removed?

Reply:
This wasn't used anywhere @muhammad-ali-e - will try to capture dead code removals in a separate PR from now on